lib/elixir_boilerplate_web/errors/helpers.ex

```diff
@@ -1,43 +1,48 @@
 defmodule ElixirBoilerplateWeb.Errors.Helpers do
-  @moduledoc """
```

Comment:
Any particular reason to remove the `@moduledoc`?

Reply:
Yes, since the module now exposes a single function (`error_messages/1`) and that function is documented (with `@doc`), I couldn’t come up with a relevant `@moduledoc` :smile: